src/test/java/org/kiwiproject/registry/consul/server/ConsulHelpersTest.java

```diff
@@ -0,0 +1,40 @@
+package org.kiwiproject.registry.consul.server;
+
+import static java.util.Map.entry;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.kiwiproject.base.KiwiEnvironment;
+import org.kiwiproject.registry.util.ServiceInfoHelper;
+
+@DisplayName("ConsulHelpers")
+class ConsulHelpersTest {
+
+    @Test
+    void shouldBuildDefaultMetadataMap() {
+        var info = ServiceInfoHelper.buildTestServiceInfo();
+
+        var kiwiEnv = mock(KiwiEnvironment.class);
+
+        var now = System.currentTimeMillis();
+        when(kiwiEnv.currentTimeMillis()).thenReturn(now);
+
+        var metadata = ConsulHelpers.buildDefaultMetaDataMap(info, kiwiEnv);
+
+        assertThat(metadata).contains(
```

Comment:
Are these _all_ that should be in metadata? If so suggest using `containsOnly`

Reply:
for this yes